modules/control/control_component.cc

```diff
@@ -89,9 +93,15 @@ bool ControlComponent::Init() {
       node_->CreateReader<PadMessage>(pad_msg_reader_config, nullptr);
   CHECK(pad_msg_reader_ != nullptr);
 
-  control_cmd_writer_ =
-      node_->CreateWriter<ControlCommand>(FLAGS_control_command_topic);
-  CHECK(control_cmd_writer_ != nullptr);
+  if (!FLAGS_use_control_submodules) {
+    control_cmd_writer_ =
+        node_->CreateWriter<ControlCommand>(FLAGS_control_command_topic);
+    CHECK(control_cmd_writer_ != nullptr);
+  } else {
+    local_view_writer_ =
+        node_->CreateWriter<LocalView>(FLAGS_control_local_view_topic);
+    CHECK(local_view_writer_);
```

Comment:
It's OK to use `CHECK(A)`, `CHECK(A != nullptr)` or `CHECK_NOTNULL(A)`, but please make it consistent, especially in one file.
Personally I like the shortest version.

Reply:
Fixed. Thank you.